setup/releases/switching-channels.md: Improve docs for LTS channel (#244)
flatcar-linux/flatcar-docs: docs/setup/releases/switching-channels.md

Multiple groups can point to the same channel. The Nebraska web interface also gives an overview about the machines and their update status.

It is recommended to start Nebraska with the `-enable-syncer` flag which keeps the Stable, Beta, Alpha, and LTS channels in sync with the public server.

> The default sync interval is one hour but may be shortened (Nebraska option `-sync-interval`). You have to create the `lts-2022` and similar channels if they don't exist on your instance.

Reviewer: I never liked the form of "you have to" in documentation - I think that "you must" or "you need to" sound better.

```suggestion
The default sync interval is one hour but may be shortened (Nebraska option `-sync-interval`). You need to create the `lts-2022` and similar channels if they don't exist on your instance.
```

To specify a particular Flatcar version you want to deploy, you should not modify the `stable` *channel* because this gets synced with the public server and your changes are lost.
You should rather create a new channel and let the `stable` *group* point to it.
When using your own Nebraska update server, the `lts` group is not switched over to point a new `lts-YEAR` channel when a new channel comes out.
To migrate the machines to the new LTS major release, first create the `lts-YEAR` channel on your instance since it may not exist, wait for the syncing to pick up the latest version for the channel, and then let the `lts` group point to the new channel.

> This has to be done manually in Nebraska and offers the advantage that the `lts` group which is the default for LTS installations can be kept and no changes on the machines itself are necessary.

Reviewer: ```suggestion
This needs to be done manually in Nebraska and it offers the advantage that the `lts` group which is the default for LTS installations can be kept and no changes on the machines themselves are necessary.
```

Author: Thanks!
